Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -208,4 +178,33 @@ extension SwiftLanguageService {
       }
     }
   }
+
+  func expandMacro(macroExpansionURLData: inout MacroExpansionReferenceDocumentURLData) async throws -> String {
```

Comment:
Why does the parameter need to be `inout`? As far as I can tell, this is only called from `getReferenceDocument`, which doesn’t use the modified parameter.